build-ps/Jenkinsfile

```diff
@@ -111,7 +111,7 @@ pipeline {
                 cd \${build_dir}
                 sudo bash -x ./percona-server_builder.sh --builddir=\${build_dir}/test --install_deps=1; \
                 curl https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip -o awscliv2.zip
-                bash -x ./percona-server_builder.sh --builddir=\${build_dir}/test --repo=${GIT_REPO} --branch=${BRANCH} --rpm_release=${RPM_RELEASE} --deb_release=${DEB_RELEASE} --build_src_rpm=1
+                bash -x ./percona-server_builder.sh --builddir=\${build_dir}/test --repo=${GIT_REPO} --branch=${BRANCH} --rpm_release=${RPM_RELEASE} --deb_release=${DEB_RELEASE} --build_src_rpm=1 --gen_changelog=${RELEASE_BUILD}
```

Comment:
having gen_changelog enabled by default would ease build process as we wouldn't need to modify our build pipelines